cirq/pasqal/__init__.py

```diff
@@ -0,0 +1,12 @@
+from .pasqal_qubits import (
+    ThreeDGridQubit,)
+
+from .pasqal_device import (
```

Comment:
Remove the `.` from the front all these imports

Reply:
It throws an error without the dot. Do you have another format in mind that you want us to use?